const getRole = async (req, res) => {
  try {
  } catch (err) {
    res.status(400).json({ error: err.message });
  }
};

const authRole = async (req, res) => {
  try {
  } catch (err) {
    res.status(400).json({ error: err.message });
  }
};

export { getRole, authRole };
